lua/nvim-web-devicons/icons-default.lua

```diff
@@ -932,8 +1316,14 @@ local icons_by_file_extension = {
     cterm_color = "74",
     name = "Fsx",
   },
+  ["gcode"] = {
+    icon = "󰫴",
```

Comment:
In the absence of a specific icon let's be consistent with the other 3d files: nf-md-cube_outline

Reply:
G code are instructions to move CNC machines, not exactly a 3D file

Reply:
How about nf-md-printer_3d ?

We are avoiding letters as icons as they can be confusing next to text e.g. file names.

Reply:
I like it